### DISCUSSION TITLE
Duration of Update is 0 and charge values incomplete

### DISCUSSION THREAD

**Discussion by manuelkamp** [teslamate-org/teslamate#2173] [Q&A]
Hello,

I have some strange things here. Yesterday I started an update. The start was logged correct, but the duration was logged as 0, but it actually was around 35 minutes. How can I correct this?

In addition to that, after the update has finished, I started a charge. It did log the correct start time. But after ~1 hour it changed the logged state to "online" and did not log the complete charge process, which was ~5 hours. But the "online" state changed accordingly to the end time of the charge to "asleep".

I already tried to fix this charge with the command from the documentation (I had to use it a few months ago because the server where I run teslamate on did a restart during a charge, so I am aware of it and how to sucessfully use it).

Does anyone have an idea how to fix this, or is this probably a bug worth to report?

best regards,
kamp

**Comment by DrMichael** [comment]
> I have some strange things here. Yesterday I started an update. The start was logged correct, but the duration was logged as 0, but it actually was around 35 minutes. How can I correct this?

Where do you see, that it is 0?

**Reply by manuelkamp** [reply]
In the "Updates" Tab 0 seconds are logged. The real update duration was about 35 minutes:
![Screenshot 2021-11-22 145052](https://user-images.githubusercontent.com/48356947/142873201-52bc3f15-c4ab-492f-98b0-4bac15236f17.png)


In addition you can see on this screenshot, that there are 0 charges logged. I charged two times (the one charge which was not logged completely and another charge, logged correct) since then, so the count also should be 2 instead of 0.

**Reply by manuelkamp** [reply]
does anyone have any suggestion on how to fix the missing number of charges and missing value for avg rated range?
